Source/JavaScriptCore/heap/StructureAlignedMemoryAllocator.cpp

```diff
@@ -94,16 +95,28 @@ class StructureMemoryManager {
 public:
     StructureMemoryManager()
     {
-        static_assert(hasOneBitSet(structureHeapAddressSize));
-        uintptr_t mappedHeapSize = structureHeapAddressSize;
+        uintptr_t preferredStructureHeapSize = structureHeapAddressSize;
+        if (Options::structureHeapSizeInKB())
+            preferredStructureHeapSize = static_cast<uintptr_t>(Options::structureHeapSizeInKB()) * KB;
```

Comment:
How about setting the default value of `structureHeapSizeInKB` to `structureHeapAddressSize / KB`?